src/data/complex/exponential_bounds.lean

```diff
@@ -38,6 +38,12 @@ lt_of_lt_of_le (by norm_num) (sub_le.1 (abs_sub_le_iff.1 exp_one_near_10).2)
 lemma exp_one_lt_d9 : exp 1 < 2.7182818286 :=
 lt_of_le_of_lt (sub_le_iff_le_add.1 (abs_sub_le_iff.1 exp_one_near_10).1) (by norm_num)
 
+/--
+A looser bound than `real.exp_one_lt_d9`, but this is sometimes more efficent for numerical
+computations.
+-/
+lemma exp_one_lt_three : exp 1 < 3 := exp_one_lt_d9.trans (by norm_num)
```

Comment:
Seems like this bound is no longer used? If you want to keep it around, maybe a separate PR would be better?

Reply:
Eh, I'll just delete it, if someone else needs it it won't be too hard to add.